content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -56,11 +56,21 @@ For more information, see [Network Plugin Requirements](/docs/concepts/extend-ku
 
 ### Forwarding IPv4 and letting iptables see bridged traffic
 
-Verify that the `br_netfilter` module is loaded by running `lsmod | grep br_netfilter`. 
+Verify that the below modules are loaded:   
+Verify `br_netfilter` module is loaded by running `lsmod | grep br_netfilter`.   
+Verify `overlay` module is loaded by running `lsmod | grep overlay`. 
 
-To load it explicitly, run `sudo modprobe br_netfilter`.
+To load the modules explicitly, run:   
+`sudo modprobe br_netfilter`   
+`sudo modprobe overlay`
 
-In order for a Linux node's iptables to correctly view bridged traffic, verify that `net.bridge.bridge-nf-call-iptables` is set to 1 in your `sysctl` config. For example:
+In order for a Linux node's iptables to correctly view bridged traffic, verify that following system variables are set to 1 in your `sysctl` config: `net.bridge.bridge-nf-call-iptables`, `net.bridge.bridge-nf-call-ip6tables`, `net.ipv4.ip_forward`.
+
+To verify, run the below command to get the values of system variables:   
+`sysctl -n net.bridge.bridge-nf-call-iptables net.bridge.bridge-nf-call-ip6tables net.ipv4.ip_forward`
```

Comment:
I recommend using a <tt>```</tt> delimited block here.

Reply:
Maybe this should come _after_ the step to configure `/etc/modules-load.d/k8s.conf`?

Reply:
I'd suggest to drop `-n` command line option. Without it the command output looks more informative:
```
$ sysctl net.bridge.bridge-nf-call-iptables net.bridge.bridge-nf-call-ip6tables net.ipv4.ip_forward
net.bridge.bridge-nf-call-iptables = 1
net.bridge.bridge-nf-call-ip6tables = 1
net.ipv4.ip_forward = 1
```

Reply:
Should we also explain how to set these variables explicitly from the command line with `sysctl -w`? This would make it consistent with the `insmod` instructions for modules.

Reply:
Thanks for the comments.
I have incorporated the comments, please let me know your views.